freqtrade/constants.py

```diff
@@ -126,6 +126,8 @@
                 'name': {'type': 'string'},
                 'key': {'type': 'string'},
                 'secret': {'type': 'string'},
+                'password': {'type': 'string'},
+                'uid': {'type': 'string'},
```

Comment:
accodring to ccxt
https://github.com/ccxt/ccxt/blob/master/python/ccxt/gdax.py#L59-L62
the required credentials are `apiKey `, `secret `, `password`, do we really need the `uid?`

Reply:
thanks for the info, no need to remove